src/lib.rs

```diff
@@ -8,6 +8,11 @@
 //! are supported with the
 //! [`Db::open_tree`](struct.Db.html#method.open_tree) method.
 //!
+//! Sled is not designed to be used by multiple processes simultaneously.
+//! Although Sled tries to use advisory filesystem locking to
+//! protect the database from accidental access from foreign
+//! processes, this should not be reiled upon.
```

Comment:
no need for this comment here

Reply:
Where should it be instead?

I don't see other mentions of the shared multiprocess use cases in the documentation, yet questions do arise.

Reply:
I've reworded this and placed it high up on the main README https://github.com/spacejam/sled/commit/66cf95758faac3cfbf837796c4cf35aa926f3ea3